### PR TITLE
Add usergroup model

### LINKED ISSUE #8: Create Group model

### LINKED ISSUE #9: Create userGroup model

### PR DESCRIPTION
**Pull request by HacksForHugs**
finished group model and user-group model
Fix #9
Fix #8

### CODE REVIEW

**Comment by HacksForHugs** [comment]
@lilshim I finished (:

File: test/fixtures/groups.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Read about fixtures at http://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
+
```
Comment (hidesminimally) [inline test/fixtures/groups.yml]: @nikitr Try not to include auto-generated files. Remove if nothing useful is added

File: test/fixtures/user_groups.yml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# Read about fixtures at http://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
+
+# This model initially had no columns defined.  If you add columns to the
+# model remove the '{}' from the fixture names and add the columns immediately
+# below each fixture, per the syntax in the comments below
+#
+one: {}
+# column: value
+#
+two: {}
```
Comment (hidesminimally) [inline test/fixtures/user_groups.yml]: @nikitr Try not to include auto-generated files. Remove if nothing useful is added

File: test/models/group_test.rb [hunk 3]
```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class GroupTest < ActiveSupport::TestCase
```
Comment (hidesminimally) [inline test/models/group_test.rb]: @nikitr Try not to include auto-generated files. Remove if nothing useful is added

File: test/models/user_group_test.rb [hunk 4]
```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
```
Comment (hidesminimally) [inline test/models/user_group_test.rb]: @nikitr Try not to include auto-generated files. Remove if nothing useful is added

File: test/fixtures/groups.yml [hunk 5]
```diff
@@ -0,0 +1,7 @@
...
+one:
+  name: MyString
```
Comment (hidesminimally) [inline test/fixtures/groups.yml]: @HacksForHugs  Try not to include auto-generated files. Remove if nothing useful is added

File: test/fixtures/user_groups.yml [hunk 6]
```diff
@@ -0,0 +1,11 @@
+# Read about fixtures at http://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
```
Comment (hidesminimally) [inline test/fixtures/user_groups.yml]: @HacksForHugs delete this

File: test/models/group_test.rb [hunk 7]
```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
```
Comment (hidesminimally) [inline test/models/group_test.rb]: @HacksForHugs this too

File: test/models/user_group_test.rb [hunk 8]
```diff
@@ -0,0 +1,7 @@
...
+
```
Comment (hidesminimally) [inline test/models/user_group_test.rb]: @HacksForHugs same

**Comment by hidesminimally** [comment]
LGTM

**Pull request merged**